### PR TITLE
CGE 50 product page fix

### PR DESCRIPTION
**Pull request by SergeyCheremisin**

### CODE REVIEW

**Review by Wolshebnik — APPROVED** [review]

File: src/main/resources/react/src/components/local/en.js [hunk 1]
```diff
@@ -50,7 +50,8 @@ const en = {
 		addField: 'Add field',
 		removeField: 'Remove field',
 		saveChanges: 'Save changes',
-		enter: 'Enter'
+		enter: 'Enter',
+		look: 'To view',
 	},
```
Comment (Jack11M) [inline src/main/resources/react/src/components/local/en.js:55]: I would prefer to change 'to view' to 'View'

File: src/main/resources/react/src/components/local/en.js [hunk 2]
```diff
@@ -93,7 +94,8 @@ const en = {
 		age: 'Age',
 		season: 'Season',
 		sex: 'Sex',
-		button: 'Offer an exchange'
+		button: 'Offer an exchange',
```
Comment (Jack11M) [inline src/main/resources/react/src/components/local/en.js:97]: As for me, it is better to use 'exchange offers' or 'offers for exchange' here

File: src/main/resources/react/src/components/local/en.js [hunk 3]
```diff
@@ -50,7 +50,8 @@ const en = {
 		addField: 'Add field',
 		removeField: 'Remove field',
 		saveChanges: 'Save changes',
-		enter: 'Enter'
+		enter: 'Enter',
+		look: 'To view',
```
Comment (Jack11M) [inline src/main/resources/react/src/components/local/en.js:54]: I would prefer to change to 'View'

File: src/main/resources/react/src/components/local/en.js (hunk 2 above)
Comment (Jack11M) [inline src/main/resources/react/src/components/local/en.js:97]: As for me, it is better to use 'Exchange offers' or 'Offers for exchange' here

**Review by Jack11M — APPROVED** [review]

**Pull request merged**